server/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.8-slim-buster
+
+#RUN groupadd -r scratchy && useradd -m -g scratchy scratchy
+#USER scratchy
+
+WORKDIR /app
+
+COPY requirements.txt requirements.txt
+RUN pip3 install -r requirements.txt
+
+COPY . .
+RUN pip3 install -e .
```

Comment:
what is that ?

Reply:
from what i understand it should be RUN pip3 install -e -r requirements.txt

Reply:
from what i understand it should be RUN pip3 install -e -r requirements.txt

Reply:
with pip3 doc  the argument -e --> ` -e,--editable <path/url>
 Install a project in editable mode (i.e.  setuptools "develop mode") from a local project path or a VCS url.
`

Reply:
yes but if we install like that there is no need for line 9, it will just add a useless image in the process

Reply:
- Line 9 : create a docker layer with dependencies, reused if dependencies doesn't change, that the good idea behind it
- Line 10 : create a docker layer with the installed app package (scratchy-server), will be rebuilt if source file change, but doesn't need the dependency layer to be rebuilt